eng/Versions.props

```diff
@@ -89,25 +89,25 @@
     <MicrosoftVisualStudioCodingConventionsVersion>1.1.20180503.2</MicrosoftVisualStudioCodingConventionsVersion>
     <MicrosoftVisualStudioComponentModelHostVersion>16.0.198-g52de9c2988</MicrosoftVisualStudioComponentModelHostVersion>
     <MicrosoftVisualStudioCompositionVersion>15.5.23</MicrosoftVisualStudioCompositionVersion>
-    <MicrosoftVisualStudioCoreUtilityVersion>16.1.15</MicrosoftVisualStudioCoreUtilityVersion>
+    <MicrosoftVisualStudioCoreUtilityVersion>16.1.32-g08f21260a1</MicrosoftVisualStudioCoreUtilityVersion>
```

Comment:
this was my private build, use 16.1.28-g2ad4df7366 - that's the official build already inserted into lab/d16.1stg